services/gitlab/gitlab-issues.service.js

```diff
@@ -0,0 +1,446 @@
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: { gitlab_url: 'https://jihulab.com' },
+      staticPreview: {
+        label: 'issues',
+        message: '611 closed',
+        color: 'yellow',
+      },
+      documentation: customDocumentation,
+    },
+    {
+      title: 'GitLab (self-managed) closed issue',
+      pattern: 'closed-raw/:project+',
+      namedParams: {
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: { gitlab_url: 'https://jihulab.com' },
+      staticPreview: {
+        label: 'closed issues',
+        message: '611',
+        color: 'yellow',
+      },
+      documentation: customDocumentation,
+    },
+    {
+      title: 'GitLab (self-managed) closed issues by-label',
+      pattern: 'closed/:project+',
+      namedParams: {
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: {
+        gitlab_url: 'https://jihulab.com',
+        labels: 'backend,feature::enhancement',
+      },
+      staticPreview: {
+        label: 'backend,feature::enhancement issues',
+        message: '57 closed',
+        color: 'yellow',
+      },
+      documentation: customDocumentation + labelDocumentation,
+    },
+    {
+      title: 'GitLab (self-managed) closed issues by-label',
+      pattern: 'closed-raw/:project+',
+      namedParams: {
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: {
+        gitlab_url: 'https://jihulab.com',
+        labels: 'backend,feature::enhancement',
+      },
+      staticPreview: {
+        label: 'closed backend,feature::enhancement issues',
+        message: '57',
+        color: 'yellow',
+      },
+      documentation: customDocumentation + labelDocumentation,
+    },
+    {
+      title: 'GitLab (self-managed) all issue',
+      pattern: 'all/:project+',
+      namedParams: {
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: { gitlab_url: 'https://jihulab.com' },
+      staticPreview: {
+        label: 'issues',
+        message: '969 all',
+        color: 'yellow',
+      },
+      documentation: customDocumentation,
+    },
+    {
+      title: 'GitLab (self-managed) all issue',
+      pattern: 'all-raw/:project+',
+      namedParams: {
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: { gitlab_url: 'https://jihulab.com' },
+      staticPreview: {
+        label: 'all issues',
+        message: '969',
+        color: 'yellow',
+      },
+      documentation: customDocumentation,
+    },
+    {
+      title: 'GitLab (self-managed) all issues by-label',
+      pattern: 'all-raw/:project+',
+      namedParams: {
+        project: 'gitlab-cn/gitlab',
+      },
+      queryParams: {
+        gitlab_url: 'https://jihulab.com',
+        labels: 'backend,feature::enhancement',
+      },
+      staticPreview: {
+        label: 'all backend,feature::enhancement issues',
+        message: '84',
+        color: 'yellow',
+      },
+      documentation: customDocumentation + labelDocumentation,
+    },
```

Comment:
I feel like we should try to cut back on the instances of example listings. Would it be possible to cut this list in half, and just add the instance url parameter on every example listing with an appropriate default, instead of having 2 of each scenario?

Reply:
Indeed